man/ctags-lang-elm.7.rst.in

```diff
@@ -0,0 +1,152 @@
+	Maybe	input.elm	/^  , Maybe$/;"	t	module:MyMod	roles:imported
+	Result	input.elm	/^  , Result(..)$/;"	t	module:MyMod	roles:imported
+	MyList	input.elm	/^  , MyList(Empty)$/;"	t	module:MyMod	roles:imported
+	Empty	input.elm	/^  , MyList(Empty)$/;"	c	type:MyMod.MyList	roles:imported
+
+Namespaces
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Namespaces are tagged and their role is "def".
+
+.. code-block:: Elm
+
+	module AMod exposing (..)
+	
+	import MyImport as NSpace exposing (impFunc)
+
+"output.tags"
+with "--sort=no --extras=+r --fields=+r"
+
+.. code-block:: tags
+
+	AMod	input.elm	/^module AMod exposing (..)$/;"	m	roles:def
+	NSpace	input.elm	/^import MyImport as NSpace exposing (impFunc)$/;"	n	module:AMod	roles:def	moduleName:MyImport
+	MyImport	input.elm	/^import MyImport as NSpace exposing (impFunc)$/;"	m	roles:imported
+	impFunc	input.elm	/^import MyImport as NSpace exposing (impFunc)$/;"	f	module:MyImport	roles:imported
+
+Type descriptions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Constructors will have type descriptions. So will any top level function
+that has a type annotation.
+
+.. code-block:: Elm
+
+	funcA : Int -> Int
+	funcA a = a + 1
+	
+	type B
+	    = B1Cons
+	      { x : Float
+	      , y : Float
+	      }
+	    | B2Cons String Integer
+	    | B3Cons
+
+"output.tags"
+with "--sort=no --extras=+r --fields=+r"
+
+.. code-block:: tags
+
+	funcA	input.elm	/^funcA a = a + 1$/;"	f	typeref:description:Int -> Int	roles:def
+	B	input.elm	/^type B$/;"	t	roles:def
+	B3Cons	input.elm	/^    | B3Cons$/;"	c	type:B	typeref:description:	roles:def
+	B2Cons	input.elm	/^    | B2Cons String Integer$/;"	c	type:B	typeref:description:String -> Integer -> B	roles:def
+	B1Cons	input.elm	/^    = B1Cons$/;"	c	type:B	typeref:description:{ x : Float , y : Float } -> B	roles:def
+
+Function parameter lists
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Function parameter lists can be extracted into the tags file
+signature field. They are not really function signatures, but
+it's the closest concept available in ctags.
+Use "--fields=+S".
+
+.. code-block:: Elm
+
+    funcA a1 a2 =
+        a1 + a2
+
+"output.tags"
+with "--sort=no --extras=+r --fields=+rS"
+
+.. code-block:: tags
+
+    funcA	input.elm	/^funcA a1 a2 =$/;"	f	signature:a1 a2	roles:def
+
+KNOWN LIMITATIONS
+-----------------
+The ctags signature field is used for function parameter lists, even
+though it's not an idea field. See above.
+
+Elm requires all statements at the same logical level to have the
+same indentation. If there is additional indentation that line is part
+of the previous one. Therefore without over-complicating the
+PEG parser we have the following limitations...
+
+Sometimes functions in let/in blocks will be omitted.
+
+Functions in let/in blocks will be marked as being in the scope of their
+outer function, regardless of how deeply nested the let/in block is.
+
+Functions in let/in blocks won't have type descriptions.
+
+SEE ALSO
+--------
+ctags(1), ctags-client-tools(7)
```

Comment:
Sorry, I took mistook. The comment about man-tests should be applied here. 
docs/man/ctags-lang-elm.7.rst is generated from this file. man/ctags-lang-elm.7.rst.in the original.